docs/helpers/SeleniumWebdriver.md

```diff
@@ -36,13 +36,13 @@ This helper should be configured in codecept.json or codecept.conf.js
 -   `browser` - browser in which perform testing
 -   `driver` - which protractor driver to use (local, direct, session, hosted, sauce, browserstack). By default set to 'hosted' which requires selenium server to be started.
 -   `restart` - restart browser between tests (default: true).
--   `smartWait`: (optional) **enables SmartWait**; wait for additional milliseconds for element to appear. Enable for 5 secs: "smartWait": 5000
+-   `smartWait`: (optional) enables SmartWait; wait for additional milliseconds for element to appear. Enable for 5 secs: "smartWait": 5000
 -   `disableScreenshots`   - don't save screenshot on failure
 -   `uniqueScreenshotNames`   - option to prevent screenshot override if you have scenarios with the same name in different suites
 -   `keepBrowserState`   - keep browser state between tests when `restart` set to false.
--   `keepCookies`   - keep cookies between tests when `restart` set to false.\*
+-   `keepCookies`   - keep cookies between tests when `restart` set to false.\
 -   `seleniumAddress` - Selenium address to connect (default: [http://localhost:4444/wd/hub][7])
--   `waitForTimeout`: (optional) sets default wait time in _ms_ for all `wait*` functions. 1000 by default;
```

Comment:
Should `*` be removed ?(5)

Reply:
this helper is deprecated, not even shown on website :)